src/EngineManager.php

```diff
@@ -138,6 +140,30 @@ protected function ensureMeilisearchClientIsInstalled()
         throw new Exception('Please install the suggested Meilisearch client: meilisearch/meilisearch-php.');
     }
 
+    /**
+     * @return TypesenseEngine
+     * @throws \Typesense\Exceptions\ConfigError
+     */
```

Comment:
Please use FQN everywhere.

```suggestion
    /**
     * Create a Typesense engine instance.
     * 
     * @return \Laravel\Scout\Engines\TypesenseEngine
     *
     * @throws \Typesense\Exceptions\ConfigError
     */
```